community/plugins/spring-ai-alibaba-starter-plugin-feishu/pom.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.1.1</version>
+		<relativePath/>
+	</parent>
+	<groupId>com.alibaba.cloud.ai.plugin.feishu</groupId>
+	<artifactId>spring-ai-alibaba-starter-plugin-feishu</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>spring-ai-alibaba-starter-plugin-feishu</name>
+	<description>FeiShu Tool for Spring AI Alibaba</description>
+
+	<properties>
+		<java.version>17</java.version>
+	</properties>
+
+	<dependencyManagement>
```

Comment:
According to the transitive relationship of mvn, after introducing the above parent, there is no need to introduce it again here.

Reply:
ok